### DISCUSSION TITLE
Welcome to oauth2-server Discussions!

### DISCUSSION THREAD

**Discussion by jankapunkt** [leaonline/oauth2-server#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by StorytellerCZ** [comment]
Hello everyone I'm Storyteller,
I work with @jankapunkt on @Meteor-Community-Packages and in the near future plan to to implement the @meteor version of the oauth2 server into @LiteraryUniverse. So for right now I'm just a lurker.
